app/schemas/schema.js

```diff
@@ -1,112 +1,230 @@
 const schema = {
-  "title": "Request to Participate",
   "type": "object",
   "required": [
-    "organization",
-    "streetNumber",
-    "streetName",
-    "city",
-    "postalCode"
+    // "Organization",
+    // "Registration Number",
+    // "Street Number",
+    // "Street Name",
+    // "City",
+    // "Postal Code",
+    // "Primary Contact",
+    // "Postion",
+    // "Contact Email",
+    // "Telephone",
+    // "Zone Feedback",
+    // "Wired Broadband",
+    // "Fixed Wireless",
+    // "Last Mile Support",
+    // "Anticipated Projects",
+    // "Underserved Areas",
+    // "Satellite Proposal",
+    // "Capital Cost Funding",
+    // "Capital Cost Explanation",
+    // "Last Mile Broadband",
+    // "Last Mile Fixed Wireless",
+    // "Last Mile New Backbone",
+    // "Highway Cellular"
```

Comment:
Why are the required fields commented out?

Reply:
oh darn, that was for testing. I'll fix that.